test/DynamoCoreTests/IconsTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Dynamo.Models;
+using Dynamo.Search.SearchElements;
+using Dynamo.UI;
+using Dynamo.Wpf.Services;
+using DynamoShapeManager;
+using DynamoUtilities;
+
+using NUnit.Framework;
+
+using System;
+using System.Collections;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+
+namespace Dynamo.Tests
+{
+    [TestFixture]
+    internal class IconsTests
+    {
+        private static string geometryFactoryPath = "";
+        private IEnumerable searchEntries;
+
+        static readonly string AssemblyPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+        static readonly string AnyCPUPath = Directory.GetParent(AssemblyPath).FullName;
+        static readonly string binPath = Directory.GetParent(AnyCPUPath).FullName;
+        static readonly string DynamoPath = Directory.GetParent(binPath).FullName;
+
+        static readonly string ResourcePath = Path.Combine(DynamoPath, @"src\Resources");
+
+        [SetUp]
+        public void PreloadShapeManager()
+        {
+            var exePath = Assembly.GetExecutingAssembly().Location;
+            var rootFolder = Path.GetDirectoryName(exePath);
+
+            var versions = new[]
+            {
+                LibraryVersion.Version219,
+                LibraryVersion.Version220,
+                LibraryVersion.Version221
+            };
+
+            var preloader = new Preloader(rootFolder, versions);
+            preloader.Preload();
```

Comment:
The `preloader.Preload()` method will cause ASM binaries to be loaded, but that's not needed for these tests, so please remove that `Preload` call.